cmake/KokkosConfigCommon.cmake.in

```diff
@@ -89,3 +89,73 @@ function(kokkos_check)
     set(${KOKKOS_CHECK_RETURN_VALUE} ${KOKKOS_CHECK_SUCCESS} PARENT_SCOPE)
   endif()
 endfunction()
+
+# this function is provided to easily select which files use nvcc_wrapper:
+#
+#       GLOBAL      --> all files
+#       TARGET      --> all files in a target
+#       SOURCE      --> specific source files
+#       DIRECTORY   --> all files in directory
+#       PROJECT     --> all files/targets in a project/subproject
+#
+FUNCTION(kokkos_compilation)
+    CMAKE_PARSE_ARGUMENTS(COMP "GLOBAL;PROJECT" "" "DIRECTORY;TARGET;SOURCE" ${ARGN})
+
+    # search relative first and then absolute
+    SET(_HINTS ${CMAKE_CURRENT_LIST_DIR}/../.. @CMAKE_INSTALL_PREFIX@)
+
+    # find kokkos_launch_compiler
+    FIND_PROGRAM(Kokkos_COMPILE_LAUNCHER
+        NAMES           kokkos_launch_compiler
+        HINTS           ${_HINTS}
```

Comment:
I added the hints so it would default to the launcher in the installation but I didn't want to hardcode anything in the event installations got moved around. That's usually not much of a concern of course but ignoring it has basically traumatized me bc of all the issues it caused with PyPi installations in timemory (the Python package I use to invoke CMake from setup.py installs to a dummy dir and then reorganizes things the way Python likes it)

Reply:
Resolved?